app/views/hosts/index.html.erb

```diff
@@ -1,4 +1,2 @@
-<% if authorized? %>
-  <% title_actions multiple_actions_select, button_group(link_to_if_authorized _("New Host"), hash_for_new_host_path) %>
-<% end %>
+<% title_actions multiple_actions_select, button_group(link_to_if_authorized _("New Host"), hash_for_new_host_path) %>
```

Comment:
Should this now be display_link_if... so it doesn't show as greyed out if not authorised?